index.src.html

```diff
@@ -563,37 +597,81 @@ <h3 id="integration-websockets">Integration with WebSockets</h3>
 
   <h3 id="integration-html">Integration with HTML</h3>
 
-  To support the checks in [[FETCH]], we store an
-  <dfn attribute for="Document">address space</dfn> property on
-  {{Document}} objects and a similar
-  <dfn attribute for="WorkerGlobalScope">address space</dfn> on
-  {{WorkerGlobalScope}} objects, which is set as follows during {{Document}} and
-  {{Worker}} initialization:
-  
-  1.  Set {{Document/address space}} to "`local`" if the resource used to
-      instantiate the {{Document}} or {{Worker}} was delivered from a
-      [=local address=], or from a URL whose [=url/scheme=] is "`file`" (see
-      [[#file-url]]).
+  To support the checks in [[FETCH]], user agents must remember the source
+  [=/address space=] of contexts in which network requests are made. To this
+  effect, the [[HTML]] specification is patched as follows:
+
+  1.  An [=/address space=] property is persisted on a number of objects:
```

Comment:
Perhaps renaming this to "IP address space" would be reasonable as well for consistency?

Reply:
That's true. My thinking was that at the top level "address space" is a bit vague, but within the context of Document and WorkerGlobalScope, the additional disambiguation was not needed. Now I'm not sure. I'll try to change it and see how that feels.

Reply:
Done.